app/mailers/user_mailer.rb

```diff
@@ -2,12 +2,12 @@
 
 # app/mailers/user_mailer.rb
 class UserMailer < ApplicationMailer
-  default from: 'notifications@example.com'
+  default from: 'suryacherukuri999@gmail.com'
```

Comment:
Not sure about this. I think it's better we send email from the event orignaizer email. That is, get the email from the user model, but I don't know if it is possible or not.